src/styles/_helpers.scss

```diff
@@ -1,12 +1,12 @@
 .rochester {
-  font-family: $rochester;
-  color: $warning;
-  font-size: 3rem;
+	font-family: "Rochester", cursive;
+	color: $warning;
+	font-size: 2.5rem;
```

Comment:
`.rochester` should not have a `font-size` or a `color` property. Just the font font-family is needed